### ISSUE TITLE
Add option to disable Main Window, and keep only Taskbar Window.

### ISSUE THREAD

**Issue by skri11-2k** [zhongyang219/TrafficMonitor#852]
I have to manually disable the Main Window everytime I boot my PC as I don't need the Main Window.

**Comment by DeliciousLiMiao** [comment]
This is machine translation. I'm glad I was able to help you.You need to right-click the main window to close the options in the figure.  [https://sm.ms/image/7sFD6EyTagexNVt](url)

**Comment by soloopooo** [comment]
> This is machine translation. I'm glad I was able to help you.You need to right-click the main window to close the options in the figure. `https://sm.ms/image/7sFD6EyTagexNVt`

please use raw-github-storage to upload the pics, you can use `Ctrl+V` to directly paste it or use markdown code `![QQ图片20210929215628.png](https://i.loli.net/2021/09/29/7sFD6EyTagexNVt.png)`
![QQ图片20210929215628.png](https://i.loli.net/2021/09/29/7sFD6EyTagexNVt.png)

**Comment by DeliciousLiMiao** [comment]
> > 这是机器翻译。 我很高兴我能够帮助你。 你需要右键单击主窗口关闭图中的选项。
> > `https://sm.ms/image/7sFD6EyTagexNVt
> > `
> 
> 请使用raw-github-storage上传照片,你可以使用 `Ctrl + V `直接粘贴或使用减价的代码 `QQ !(图20210929215628.片png) (https://i.loli.net/2021/09/29/7sFD6EyTagexNVt.png) ` ![QQ图片20210929215628.png](https://camo.githubusercontent.com/938f7474a64688c79dc4a27bc8b905c88f5f25352d3a326146c5b88f5d8f790e/68747470733a2f2f692e6c6f6c692e6e65742f323032312f30392f32392f3773464436457954616765784e56742e706e67)

卧槽 ==  我才知道

**Comment by soloopooo** [comment]
> > > 这是机器翻译。 我很高兴我能够帮助你。 你需要右键单击主窗口关闭图中的选项。
> > > `https://sm.ms/image/7sFD6EyTagexNVt `
> > 
> > 
> > 请使用raw-github-storage上传照片,你可以使用 `Ctrl + V `直接粘贴或使用减价的代码 `QQ !(图20210929215628.片png) (https://i.loli.net/2021/09/29/7sFD6EyTagexNVt.png) ` ![QQ图片20210929215628.png](https://camo.githubusercontent.com/938f7474a64688c79dc4a27bc8b905c88f5f25352d3a326146c5b88f5d8f790e/68747470733a2f2f692e6c6f6c692e6e65742f323032312f30392f32392f3773464436457954616765784e56742e706e67)
> 
> 卧槽 == 我才知道

老兄 你quote reply的时候能不能把浏览器的翻译关了（

**Comment by DeliciousLiMiao** [comment]
> > > > 这是机器翻译。 我很高兴我能够帮助你。 你需要右键单击主窗口关闭图中的选项。
> > > > `https://sm.ms/image/7sFD6EyTagexNVt `
> > > 
> > > 
> > > 请使用raw-github-storage上传照片,你可以使用 `Ctrl + V `直接粘贴或使用减价的代码 `QQ !(图20210929215628.片png) (https://i.loli.net/2021/09/29/7sFD6EyTagexNVt.png) ` ![QQ图片20210929215628.png](https://camo.githubusercontent.com/938f7474a64688c79dc4a27bc8b905c88f5f25352d3a326146c5b88f5d8f790e/68747470733a2f2f692e6c6f6c692e6e65742f323032312f30392f32392f3773464436457954616765784e56742e706e67)
> > 
> > 
> > 卧槽 == 我才知道
> 
> 老兄 你quote reply的时候能不能把浏览器的翻译关了（

哈哈哈 ，不好意思 我开了网页自动翻译

**Comment by skri11-2k** [comment]
yeah, I get this, but the problem being that machine remembers this setting for only one session, the next time I boot up my PC, the main window pops up on startup.
All I want is the taskbar window to run on startup, not the main window.

**Comment by DeliciousLiMiao** [comment]
> yeah, I get this, but the problem being that machine remembers this setting for only one session, the next time I boot up my PC, the main window pops up on startup. All I want is the taskbar window to run on startup, not the main window.

Maybe you need to reinstall ？It didn't have a problem saving the configuration on my computer😂

**Comment by soloopooo** [comment]
> yeah, I get this, but the problem being that machine remembers this setting for only one session, the next time I boot up my PC, the main window pops up on startup. All I want is the taskbar window to run on startup, not the main window.

oh this is a point. what you need to do is, after setting, exit the program manually and then re-open it but not close it with your computer shut down.
